eks-distro-base/Dockerfile.minimal-base-python

```diff
@@ -20,15 +21,95 @@
 ARG BASE_IMAGE=unused
 ARG BUILDER_IMAGE=unused
 ARG BUILT_BUILDER_IMAGE=unused
+ARG MINIMAL_BASE_BUILDER_IMAGE=unused
 ARG VERSIONED_VARIANT
-FROM ${BUILDER_IMAGE} as builder-python-base
+ARG AL_TAG
+ARG COMPILER_VERSION=${VERSIONED_VARIANT}
+ARG IS_COMPILER=false
+
+
+################# COMPILE ON GLIBC ######################
+# Compile in the minimal base builder  image to allow sharing result across compiler
+# and runtime images. We also build the cache into the runtime image
+# which includes the cache data for this layer and along with COPY --link
+# this also us to ensure this layer is always reusable across the runtime + compiler
+# builds without relying on the buildkitd local cache
+FROM ${MINIMAL_BASE_BUILDER_IMAGE} as python-compile-3.9-al2
```

Comment:
Since it's set to `unused` above, how does this work? I see this in a couple places in this file.

Reply:
Just ignored the word `unused`, thats a bad default. It is def used but due to a quirk with how docker parses these files, if that var isnt passed because its not needed for the given stage, it will still barf. Setting to something (anything) avoids it from bailing out.